chart/templates/_postgresql.tpl

```diff
@@ -2,5 +2,9 @@
 Name of the postgresql clusterIP service
 */}}
 {{- define "xnat.postgresql.svc" -}}
+{{- if .Values.postgresql.fullnameOverride -}}
+{{- printf "%s-rw" .Values.postgresql.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
```

Comment:
set using the `fullnameOverride` as to match the name of the service created by CNPG